src/Main.hs

```diff
@@ -147,6 +174,73 @@ checkSession db = do
         Just session -> return session
         Nothing -> raiseStatus status401 "Invalid session cookie"
     Nothing -> raiseStatus status401 "Invalid session cookie"
+data Ingredient = Ingredient {count :: Int, ingredient :: String} deriving (Read, Show, Generic)
```

Comment:
`count` ist nicht immer angemessen. Hier sollte in einem zukünftigen MR verschiedene Units supported werden (grams, mililiters, none)

Reply:
Not a part of this merge request, but rather a feature that can be implemented in the future. Not resolved by 9d8ac164eaaf6e8355e1254ae071ac5eb29b6aef